gateway/ln-gateway/src/lib.rs

```diff
@@ -76,6 +77,13 @@ impl LnGateway {
         receiver: mpsc::Receiver<GatewayRequest>,
         task_group: TaskGroup,
     ) -> Self {
+        //tokio::time::sleep(Duration::from_secs(15)).await;
```

Comment:
Leftover.

Reply:
> This is pretty much ready except for the [WASM incompatibility](https://github.com/fedimint/fedimint/actions/runs/4001915458/jobs/6868639875) and maybe some clippy nits. I don't know the WASM ecosystem that well, what would you suggest to use for `SystemTime` @Maan2003? Does [`wasm_timer`](https://docs.rs/wasm-timer/latest/wasm_timer/struct.SystemTime.html) sound good?

`Duration::from_secs_f64(js_sys::Date::new_0().get_time() / 1000.)` is used when we generate invoices